backend/settings/base.py

```diff
@@ -127,3 +127,14 @@ def get_env_variable(var_name, default):
 STATICFILES_DIRS = (
     os.path.join(BASE_DIR, 'static'),
 )
+
+REST_FRAMEWORK = {
+    
+    'DEFAULT_AUTHENTICATION_CLASSES': (
```

Comment:
@Par010 why are we using jwt!!!! 👿

Reply:
that's my mistake. I had removed it in the next commit but forgot to push the changes